cluster/addons/calico-policy-controller/calico-node-vertical-autoscaler-deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      priorityClassName: system-node-critical
```

Comment:
This should be `system-cluster-critical` as it can be moved to another node if necessary.